build/maven.go

```diff
@@ -176,12 +179,16 @@ func (mm *MavenModule) loadMavenHome() (mavenHome string, err error) {
 		// We need to grab the location using the mvn --version command
```

Comment:
it is difficult because the result of loadMavenHome is based on an operating system and we don't have control on it
it's also redundant because the function is an is-else logic